submissions/denismaysov/html-css-popup/index.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
You set the main language of your page as "English" but you use russian. Also, try to check your text on typos
![image](https://user-images.githubusercontent.com/16196199/187064456-0232b358-f25d-4a92-973f-cd39c6dc8672.png)